packages/web/src/components/gcds-radios/gcds-radios.tsx

```diff
@@ -350,7 +416,6 @@ export class GcdsRadios {
 
     this.validateValidator();
 
-    this.inheritedAttributes = inheritAttributes(this.el, this.shadowElement);
```

Comment:
Is this not needed anymore? I'm trying to understand why we need to remove this line (same for checkboxes)

Reply:
Currently the inheritedAttributes is an experimental feature that isn't documented anywhere. In it's current form it can only handle one element, and with these two components being groups of multiple elements, it would require a refactor. I just don't think it's worth it at this time to look into that.